orchestra/static/orchestra/project-management/js/crosshair.es6.js

```diff
@@ -0,0 +1,64 @@
+export default function crosshair (visUtils, axis) {
```

Comment:
I was hesitant to move things around since this code is more complicated, but now that it's all working again I'll flatten the structure.